android/app/src/test/java/net/pengcook/android/data/repository/auth/FakeAuthorizationRepository.kt

```diff
@@ -81,4 +81,8 @@ class FakeAuthorizationRepository(
     override suspend fun fetchUserInformation(accessToken: String): Result<UserInformation> {
         return runCatching { userInformation ?: throw IllegalStateException() }
     }
+
+    override suspend fun deleteAccount(): Result<Unit> {
+        TODO("Not yet implemented")
+    }
```

Comment:
대충 아래와 같은 형태로라도 구현은 해놓는게 좋을 것 같습니다!
```suggestion
    override suspend fun deleteAccount(): Result<Unit> {
        return runCatching { 
            Unit // 혹은 생략도 가능
        }
    }
```